recipe/meta.yaml

```
@@ -0,0 +1,101 @@
+{% set name = "recommenders" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/recommenders/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8db3785543b2ace6ddc68994e71c8920efa781432708d8d4cc7aff720f11b55f
+
+build:
+  number: 0
+  skip: True  # [py<38 or py>=310 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+    - numpy 1.24.3
+  run:
+    - bottleneck >=1.2.1,<2
+    - category_encoders >=1.3.0,<2
+    - cornac >=1.1.2,<2
+    - jinja2 >=2,<3.1
+    - lightfm >=1.15,<2
+    - lightgbm >=2.2.1
+    - matplotlib >=2.2.2,<4
```

```suggestion
    - matplotlib-base >=2.2.2,<4
```
To not pull in Qt unnecessarily